tests/InternalClient.php

```diff
@@ -84,10 +84,6 @@ public function getUserID() {
     public function setUserID($userID) {
         $this->userID = $userID;
 
-        /* @var \Gdn_Session $session */
-        $session = $this->container->get(\Gdn_Session::class);
-        $session->start($userID, false, false);
-
```

Comment:
I recommend keeping this part in here. The function allowed you to change users on the current session. Now, it's just setting a property on the object and you're stuck with always being the admin user.

Reply:
This may be a premature removal but I can already tell that handling the session management there won't be enough. I'll close for now and apply the session start in setupBeforeClass and we can address that issue later then.